config/levels/config2.cfg

```diff
@@ -0,0 +1,126 @@
+level = {
+    name = "Forest Adventure",
+    description = "A peaceful forest area with simple enemies to introduce basic game mechanics.",
+    difficulty = "EASY",
+    background_music = "forest_theme.mp3",
+    cell_size = 30,
+    map_size = { x = 1000, y = 10 },
+    view_port = { x = 1920, y = 1080 },
+    goal = "Reach the exit while avoiding enemy patrols."
+},
```

Comment:
Is this file really the forest adventure ?

Reply:
Nope and it has been modified in a more recent pull request (119) :)

Reply:
okay nice